### DISCUSSION TITLE
Welcome to RustBackyard Discussions!

### DISCUSSION THREAD

**Discussion by Praying** [Praying/RustBackyard#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MATRIXKOO** [comment]
😎

**Reply by Praying** [reply]
```rust
println!("Welcome!")
```

**Comment by Pure-Peace** [comment]
wow

**Reply by Praying** [reply]
```rust
println!("Welcome Pure-Peace")
```

**Reply by MATRIXKOO** [reply]
talk thread! wow!
